packages/web3-utils/test/unit/formatter.test.ts

```diff
@@ -479,6 +479,26 @@ describe('formatter', () => {
 					).toEqual(new Uint8Array([16, 11, 202]));
 				});
 			});
+
+			describe('string', () => {
+				it('should format string for 123', () => {
+					expect(
+						format({ format: 'string' }, 123, {
+							number: FMT_NUMBER.STR,
+							bytes: FMT_BYTES.HEX,
+						}),
+					).toBe('123');
+				});
+
+				it('should format string for 0x1234', () => {
+					expect(
+						format({ format: 'string' }, 0x1234, {
```

Comment:
```suggestion
				it('should format string for 1234, () => {
					expect(
						format({ format: 'string' }, 1234, {
```
We don't actually test using hex, formatting with numbers imo would be better